### ISSUE TITLE
Option to not remove WebCdn when running in dev mode

### ISSUE THREAD

**Issue by mbhoward73** [sst/ion#813]
It appears that WebCdn is now automatically removed when running in dev mode.  The problem is that when I switch between doing a dev deploy and a regular deploy it takes about 5 minutes to do the teardown or re-creation of the Cdn.  It would be nice if the WebCdn teardown when running local mode could be disabled with config.

|  Created     Web sst:aws:SvelteKit → WebCdnWaiter sst:aws:DistributionDeploymentWaiter (260.0s)

**Comment by thdxr** [comment]
we generally do not support workflows that involving flipping stages back and forth - there's too much that is different<br><br>can you describe your use case a bit more?

**Comment by mbhoward73** [comment]
the use case is when I'm doing local development against my personal stage in dev mode (which I LOVE btw) but for various reasons I need to do something which doesn't work well in dev mode so I need to switch and do a regular deploy to my personal stage temporarily before switching back to dev mode.  I try to stay in dev mode whenever possible but I will fairly frequently hit something which doesn't work well in dev mode which prompts me to switch to a regular deploy.  here are some of the issues I've encountered with dev mode which prompt me to switch:

- I like to run our entire test suite against my personal stage before checking in my feature branch.  When running my project in dev mode, the test run is a lot slower and I frequently run into test errors which I never encounter when running against a regular deploy.  I haven't tracked down the issues yet but our test suite uses a lot of a parallelism so I suspect something related to that.  Running with pnpm workspace-concurrency to 1 seems to work more reliably but takes far too long.
- logging to the console in dev mode is frequently out of order and sometimes hard to make sense of.  sometimes I'm trying to debug something which gets too hard to track with the logging all out of order so I'll do a regular deploy to see more properly ordered logs in cloudwatch
- sometimes I'll just get my environment into some sort of bad state that I don't understand and restarting dev mode (always my first recourse) doesn't fix it.  this is admittedly vague and I haven't spent enough time tracking all these issues down but I've seen a variety of issues over the past month or so when my environment doesn't start up or a request fails or just some sort of general wierdness that I don't understand and if a dev mode restart doesn't fix it then I'll turn to a regular deploy and everything suddenly gets more stable.

**Comment by mbhoward73** [comment]
yep, I was talking about terminal output being jumbled.  regarding the other issues, yes definitely once dev mode gets more stable I'll have no reason to switch to regular deploys so frequently.  thanks for the fixes!
